lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -13,6 +13,13 @@
 from pbench.cli.agent import CliContext, pass_cli_context
 from pbench.cli.agent.commands.tools.base import ToolCommand
 from pbench.cli.agent.options import common_options
+from pbench.agent.tool_group import BadToolGroup
+
+# FIX:
```

Comment:
Seems like this is the `main()` method doc string.

Reply:
I don't understand what you mean.